lib/src/model/podcast.dart

```diff
@@ -101,6 +107,11 @@ class Podcast {
         }
       }
 
+      Value value;
+      if (rssFeed.podcastIndex != null && rssFeed.podcastIndex.value != null) {
+        value = _loadValue(rssFeed.podcastIndex.value);
```

Comment:
The method _loadValue already checks for null so we can skip the conditional check and just write:
`value = _loadValue(rssFeed?.podcastIndex?.value);`